### ISSUE TITLE
Any updates planned for TGT and LoE cards?

### ISSUE THREAD

**Issue by ConneXNL** [HearthSim/Brazier#2]
Do you have any updates planned for TGT and LoE cards?

**Comment by kelemen** [comment]
Yes, but my plan is to add it on demand, that is once I have the time to do my further plans or someone else wants it. Do you need it? What do you want to use this project for?

Adding the cards is not that much work, so if you need it for something, I can probably find some time to add those cards.

**Comment by fearofcode** [comment]
Out of curiosity, how would one implement Discover cards like Jeweled Scarab/Dark Peddler/Finley that require the player to choose from a random result set?

**Comment by kelemen** [comment]
You need some additional `EntitySelector` and `TargetlessAction` implementation. Otherwise it is similar to [Ysera](https://github.com/HearthSim/Brazier/blob/0d8b53a0309aeab121c2666d7d9ce15b16e1fca9/Brazier/cardDb/cards/ysera.card): You have to replace with `TargetlessActions.addSelectedCard` with a new method: `TargetlessActions.addPickedCard` (this method does not yet exist) and probably there should be a special `EntitySelector` implementation. You need a special implementation because cards are not chosen uniformly from a set of cards as far as I know (class specific cards have higher chance).
